DATA/UNIVERSE/SYSTEMS/EW01/ew01.ini

```diff
@@ -442,6 +442,16 @@ visit = 36
 sort = 99
 ids_info = 65909
 
+[zone]
+nickname = Zone_Ew01_sun_superdeath
+pos = 0, 0, 0
+shape = SPHERE
+size = 3000

```

Comment:
Shouldn't the death zones of these suns be bigger? Right now the normal and "superdeath" are the same.

Reply:
See the comment on that commit: the death zone of the suns in Tau 37 is tiny for their size, and increasing them isn't an option because certain patrol paths and typical player paths already are close to it. For that reason, and because the suns look very clearly big to the players, it's okay if capital ships get no "melt zone" around this one.

Reply:
But then it makes no sense adding a new zone. Just take the normal zone and make it ultimately deadly.

Reply:
Done.